desktop/src/main/java/bisq/desktop/primary/main/content/settings/userProfile/UserProfileView.java

```diff
@@ -149,14 +149,7 @@ protected void onViewAttached() {
         });
 
         selectedChatUserIdentityPin = EasyBind.subscribe(model.getSelectedUserIdentity(),
-                userIdentity -> {
-                    if (userIdentity != null) {
-                        // Setting selection via selectionModel.select displays the label from the base class. 
-                        // Did not find out how to avoid that... ;-(
-                        // With setting our selection to the editorTextField it behaves as expected.
-                        UIThread.runOnNextRenderFrame(() -> comboBox.getEditorTextField().setText(comboBox.getConverter().toString(userIdentity)));
-                    }
-                });
+                userIdentity -> comboBox.getSelectionModel().select(model.getSelectedUserIdentity().get()));
```

Comment:
`model.getSelectedUserIdentity().get()` can be replaced by the param `userIdentity`

Reply:
Yes, fixed.